### ISSUE TITLE
Fintrac not using the Drupal WET distro

### ISSUE THREAD

**Issue by LeeHunter** [wet-boew/wet-boew-drupal#497]
Sorry if this isn't the right place to post this, but I thought it was very curious and probably of interest. Fintrac is looking for a developer to create a Drupal site (bilingual, WET, WCAG etc.) and one of the specifications is that the site "not use the current Drupal WET distro available via github." 

http://www.maxhire.net/cp/searchresults.aspx?jobpreviewkey=586C361D43515B7651126539571A6D07627E4A55&AspxAutoDetectCookieSupport=1

Anybody know why that is?

**Comment by sinkinsona** [comment]
I know that Fintrac was one of the early adopters of the first Drupal WET 2.x prototype version and they did experience some issues. We offered assistance to help but never heard back from them. This being said we have since moved to WET 3.x and have made many improvements since initial release.  To not leverage the group community effort makes no sense to me as they can reuse what has been already developed instead of duplicating efforts. Perhaps they should take a look at our recent work and just upgrade instead.

**Comment by sylus** [comment]
Yeah pretty much what Andrew said. They were using an initial prototype. I have personally offered to upgrade them to the latest stable but have not heard back. They are on code that is based on a year ago's work.

To be honest though it is certainly un-motivating to hear stuff like this. Especially since they want to have integration with WET and let me be clear. There is no other best practice way to integrate with WET other then the way we have done. You have to use the Libraries API and it has to be consistent. We still have more widgets to add but the base is rock solid.

We also have amazing level of integration with WET's Intranet, Internet, and Base theme that has fully compliant WCAG + HTML5 markup. Go ahead and take a look as this was hard to accomplish and again was done in a best practice manner. So the Theme is bang on and perfected, and the WET integration is built with a strong foundation. 

I can absolutely yield to the point that the initial prototype was less then ideal. However I am extremely proud of the recent work that has been done in our currents build and think in particular our theme has an incredibly tight and impressive integration with WET.

I would have loved to hear any problems they were having and correct them. Although I would bet the recent builds solve 95% of issues. However am consoled by the fact that even Acquia itself was looking at our work ^_^

It does really get to me that based on a prototype built a year ago; a broad brush was painted that all of the work done recently can't stand on its own. 

We have a great and thriving community here and am confident that the WET Distro will only improve each day thanks to the many other departments that have started collaborating. You have to keep in mind that it is a snowball effect that the bigger the community and bugs reported and found the better the platform is in general.

**Comment by LaurentGoderre** [comment]
@sylus, I would not take it personally as in retrospect, WET versions before 3.0 also lacked maturity, maturity that could only be reached by working together and sharing our lessons learned. 

The community has stepped up and help us make both product better and this will be increasingly more obvious.

**Comment by sylus** [comment]
Yeah @LaurentGoderre completely agree. I do hope I get a chance to hear about the problems that were had. Even though the code was out of date and would love to demo the new stuff :)

I am still positive improvements could be made based on feedback though if only to improve experience.

Will just have to double efforts. Maybe even leverage more the WET Usability folks!
